README.md

```diff
@@ -90,6 +90,8 @@ MIT
 #### Version
 | Version | Build | Changes |
 | ------- | ----- | ------- |
+| **1.4.0** | **b1** | Support for CloudGenix SDK 5.5.1b1|
+|           | **b1** | Support for element step upgrade/downgrade|
```

Comment:
Minor formatting fix - lines 93/94 should read as follows:
`| **1.4.0** | **b1** | Support for CloudGenix SDK 5.5.1b1|
|           |            | Support for element step upgrade/downgrade|`

Reply:
Actually, will make a commit to fix this post merge. Disregard.